sycl/source/event.cpp

```diff
@@ -117,7 +117,14 @@ event::get_profiling_info() const {
 
 #undef __SYCL_PARAM_TRAITS_SPEC
 
-backend event::get_backend() const noexcept { return getImplBackend(impl); }
+backend event::get_backend() const noexcept {
+  try {
+    return getImplBackend(impl);
+  } catch (std::exception &e) {
+    __SYCL_REPORT_EXCEPTION_TO_STREAM("exception in get_backend", e);
+    std::abort();
+  }
+}
```

Comment:
Use try-catch syntax for the function itself, as on the example in the previous comment

Reply:
Done.